src/server/scripts/Kalimdor/TempleOfAhnQiraj/boss_skeram.cpp

```diff
@@ -84,6 +85,12 @@ class boss_skeram : public CreatureScript
                     ((TempSummon*)me)->UnSummon();
             }
 
+            void SpellHitTarget(Unit* target, SpellInfo const* spell) override
+            {
+                if (spell->Id == SPELL_TRUE_FULFILLMENT)
+                    target->CastSpell(target, SPELL_TRUE_FULFILLMENT_2, true, NULL, NULL, me->GetGUID());
```

Comment:
Trinity standards tells to use nullptr instead null :)
so please, always use standards :)